CHANGELOG.md

```diff
@@ -4,6 +4,46 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.44.0
+
+This release introduces the first alpha version of `@sentry/sveltekit`, our newest JavaScript SDK for Sveltekit. Check out the [README](./packages/sveltekit/README.md) for usage instructions and what to expect from this alpha release.
+
+- feat(browser): Export `BrowserTracing` from `@sentry/browser` (#7472)
+- feat(browser): Re-export `addTracingExtensions` from `@sentry/core` (#7498)
+- feat(bundles): Ensure CDN bundles always have a `Replay` export (#7414)
+- feat(core): Emit `beforeEnvelope` hook (#7448)
+- feat(replay): Add `request_body_size` & `response_body_size` to fetch/xhr (#7407)
+- feat(replay): Add additional properties for UI clicks (#7395)
+- feat(replay): Reduce time limit before pausing a recording (#7356)
+- feat(replay): Upgrade `rrweb` and `rrweb-player` (#7508)
+- feat(replay): Use new afterSend hook to improve error linking (#7390)
+- feat(serverless): Publish lambda layer for Node 16/18 (#7483)
+- feat(sveltekit): Add wrapper for client load function (#7447)
+- feat(sveltekit): Add wrapper for server load function (#7416)
+- feat(sveltekit): Add server-side `handleError` wrapper (#7411)
+- feat(sveltekit): Introduce client-side `handleError` wrapper (#7406)
+- feat(sveltekit): Add SvelteKit client and server `init` functions (#7408)
+- feat(sveltekit): Inject `Sentry.init` calls into server and client bundles (#7391)
+- feat(tracing): Bring back `finishReason` for interaction transactions (#7449)
+- feat(tracing): Move `registerErrorInstrumentation` to `@sentry/core` (#7494)
+- feat(tracing): Move Integrations from `tracing-internal` to `@sentry/tracing` (#7471)
+- feat(tracing): Move tracing code to internal package (#7451)
+- fix(core): Permanent idle timeout cancel finishes the transaction with the last finished child
+- fix(integrations): Handle lower-case prefix windows paths in `RewriteFrames` (#7506)
+- fix(next): Guard against missing serverSideProps (#7517)
+- fix(nextjs): Fix broken server component wrapping because of interrupted promise chain (#7456)
+- fix(nextjs): Fix runtime error for static pages (#7476)
+- fix(profiling): Catch sendProfile rejection (#7446)
+- fix(replay): Ensure network breadcrumbs have all data (#7491)
```

Comment:
```suggestion
- fix(replay): Ensure network breadcrumbs have all data (#7491)
```

We can remove this, as it actually just fixes another PR that is mentioned above!